### PR TITLE
Azure support for Dedicated

### PR DESCRIPTION
**Pull request by micheleRP**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/2669 
Review deadline: Friday, Aug 23

## Page previews
[What's New](https://deploy-preview-33--rp-cloud.netlify.app/redpanda-cloud/get-started/whats-new-cloud/)
[Create Dedicated Cluster](https://deploy-preview-33--rp-cloud.netlify.app/redpanda-cloud/get-started/cluster-types/dedicated/create-dedicated-cloud-cluster-aws/)
[Dedicated tiers and regions](https://deploy-preview-33--rp-cloud.netlify.app/redpanda-cloud/reference/tiers/dedicated-tiers/)
[Cloud Overview](https://deploy-preview-33--rp-cloud.netlify.app/redpanda-cloud/get-started/cloud-overview/)

## Checks

- [X] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/reference/partials/tiers.adoc [hunk 1]
```diff
@@ -159,5 +159,18 @@ Amazon Web Services (AWS)::
...
++
+--
+|=== 
+| Region
+
+| Central US
+| East US 
+| East US 2
+| Norway East
+| UK South
```
Comment (ncole) [inline modules/reference/partials/tiers.adoc:172]: East US
UK South
Norway East
North Europe

**Review by ncole — APPROVED** [review]
Other than region list, LGTM.  Please wait for a review from @bpraseed or product before merging.

File: modules/reference/partials/tiers.adoc (hunk 1 above)
Comment (micheleRP) [reply modules/reference/partials/tiers.adoc:171]: fixed, thanks @ncole!

File: modules/get-started/pages/whats-new-cloud.adoc [hunk 2]
```diff
@@ -6,6 +6,12 @@
 
 This page lists new features added in Redpanda Cloud.
 
+== August 2024
+
+=== Dedicated on Azure: LA
```
Comment (c4milo) [inline modules/get-started/pages/whats-new-cloud.adoc:11]: do we explain our product feature lifecycle somewhere else? some people might wonder that `LA` means.

**Review by c4milo — APPROVED** [review]

File: modules/get-started/pages/whats-new-cloud.adoc (hunk 2 above)
Comment (micheleRP) [reply modules/get-started/pages/whats-new-cloud.adoc:11]: @c4milo: In that's What's New page, we've listed other features similarly as LA or beta. In our doc pages, features in beta show a beta badge, but features in LA just show an admonition that says "to unlock this feature, contact support." An exception is [Serverless LA](https://deploy-preview-33--rp-cloud.netlify.app/redpanda-cloud/get-started/cluster-types/serverless/), where we mention the LA limits. 
image

image

**Review by bpraseed — COMMENTED** [review]
LGTM

File: modules/get-started/pages/cluster-types/dedicated/create-dedicated-cloud-cluster.adoc [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+:page-aliases: cloud:create-dedicated-cloud-cluster-aws.adoc, deploy:deployment-option/cloud/provision-a-dedicated-cluster/index.adoc, deploy:deployment-option/cloud/create-dedicated-cloud-cluster-aws.adoc, get-started:cluster-types/dedicated/create-dedicated-cloud-cluster-aws.adoc
 
 After you log in to https://cloud.redpanda.com[Redpanda Cloud^], you land on the *Clusters* page. This page lists all the clusters in your organization. 
 
 == Create a Dedicated cluster
 
 . On the Clusters page, click *Create cluster*, then click *Create Dedicated cluster*.
 +
-Enter a cluster name, then select the resource group, cloud provider (AWS or GCP), xref:reference:tiers/dedicated-tiers.adoc[region, availability zones, tier], and Redpanda version.
+Enter a cluster name, then select the resource group, cloud provider (AWS, GCP, or Azure), xref:reference:tiers/dedicated-tiers.adoc[region, tier], availability zones, and Redpanda version.
```
Comment (Feediver1) [inline modules/get-started/pages/cluster-types/dedicated/create-dedicated-cloud-cluster.adoc:11]: Do you always have to select more than one AZ? If not, then I think this should be singular: 
availability zone,

File: modules/get-started/pages/whats-new-cloud.adoc (hunk 2 above)
Comment (Feediver1) [reply modules/get-started/pages/whats-new-cloud.adoc:11]: We need to define what we mean by Limited Availability +1 to @c4milo 's comment. If not here, in another PR to follow immediately.

**Review by Feediver1 — APPROVED** [review]

File: modules/get-started/pages/cluster-types/dedicated/create-dedicated-cloud-cluster.adoc (hunk 3 above)
Comment (micheleRP) [reply modules/get-started/pages/cluster-types/dedicated/create-dedicated-cloud-cluster.adoc:11]: Correct, they should select the availability (single or multi). I've fixed!

File: modules/get-started/pages/whats-new-cloud.adoc (hunk 2 above)
Comment (micheleRP) [reply modules/get-started/pages/whats-new-cloud.adoc:11]: I'll create a new issue for this. We need product confirmation on the definition of LA.

**Pull request merged**